.github/workflows/ci.yml

```diff
@@ -13,15 +13,6 @@ jobs:
       - uses: actions/checkout@master
       - name: Check License Lines
         uses: kt3k/license_checker@v1.0.6
-  validate-locales:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js 16.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-      - run: yarn run locale:validate
```

Comment:
What does this do? Why remove it ?

Reply:
It runs a check that validates the Chinese language entries are matching and in the same order as the English language entries. We have decided earlier on that we don't want to support the Chinese language option. This was the smallest change that would allow us to work around the checks.